### ISSUE TITLE
Visual Studio git

### ISSUE THREAD

**Issue by ohhmm** [microsoft/vscode#202426]
Please, consider supporting git bundled with existing Visual Studio installation for Windows. 
No need to download & install git if it is installed with Visual Studio already.

**Comment by lszomoru** [comment]
Thank you very much for filing this issue. This request has come up over the years, but at the moment we are not planning on bundling git with VS Code so I will go ahead and close this issue as out of scope. Thank you for your understanding.

**Comment by ohhmm** [comment]
bundling git with VS Code is irrelevant to the issue.

It is about git that is already bundled with Visual Studio (not Code) installed in parallel 

Looks like misunderstanding

**Comment by lszomoru** [comment]
This can already be done through the `git.path` setting.

**Comment by ohhmm** [comment]
That is correct. And this issue is about making it for developer automatically if no other git.exe found.

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
